snuba/cli/offline_replacer.py

```diff
@@ -21,7 +21,7 @@
 @click.option(
     "--storage",
     "storage_name",
-    type=click.Choice(["errors", "errors_v2"]),
+    type=click.Choice(["errors"]),
```

Comment:
Should we just remove the parameter altogether ?

Reply:
I think explicit is good. Offline replacer should work for others too, just a coincidence there is only one today.

Reply:
I would like to keep it for now. There is no harm in having it.